build.gradle

```diff
@@ -0,0 +1,14 @@
+apply plugin: 'java'
```

Comment:
To jest ten plik gradle. Omówię co robią poszczególne rzeczy. Na początku zaznaczamy, że to ma służyć do budowania javy